src/billing-form/billing-form.component.tsx

```diff
@@ -207,36 +211,38 @@ const BillingForm: React.FC<BillingFormProps> = ({ patientUuid, closeWorkspace }
         defaultSelected="radio-1"
         className={styles.billingItem}
         onChange={toggleSearch}>
-        <RadioButton labelText={t('stockItem', 'Stock Item')} value="Stock Item" id="radio-1" />
-        <RadioButton labelText={t('service', 'Service')} value="Service" id="radio-2" />
+        <RadioButton labelText={t('stockItem', 'Stock Item')} value="Stock Item" id="stockItem" />
+        <RadioButton labelText={t('service', 'Service')} value="Service" id="service" />
       </RadioButtonGroup>
 
       <div>
         <Search
-          id="searchField"
           size="lg"
-          placeholder="Find your drugs here..."
-          labelText="Search"
+          id="searchField"
           disabled
-          closeButtonLabelText="Clear search input"
-          onChange={() => {}}
+          closeButtonLabelText={t('clearSearchInput', 'Clear search input')}
           className={styles.billingItem}
-          onKeyUp={(e) => {
-            filterItems(e.target.value);
-          }}
+          placeholder={t('findYourItemsHere', 'Find your items here')}
+          labelText={t('findYourItemsHere', 'Find your items here')}
```

Comment:
What do u think of a more direct branding like so;

```suggestion
          placeholder={t('searchItems', 'Search items and services')}
          labelText={t('searchItems', 'Search items and services')}
```